code/datums/mutantraces.dm

```diff
@@ -1391,7 +1391,7 @@
 /datum/mutantrace/ithillid
 	name = "ithillid"
 	icon_state = "squid"
-	jerk = 1
+	jerk = 0
```

Comment:
Instead of setting this to 0, it may be better to just remove this assignment completely since the default for jerk is already 0. Also, same for frogs

Reply:
missed this in review before merge but will keep it in mind if go back; was thinking about chaging the var to True/False vs 1/0 too, so I can hit it again then!